app/assets/javascripts/embed.js

```diff
@@ -0,0 +1,38 @@
+import Embed from 'flat-embed';
+
+var container = document.getElementById("embed-container");
+var embed = new Embed(container, {
+  "width": "100%",
+  "height": "450",
+  "score": "<%= @score.id %>",
+  "embedParams": {
+    "mode": "edit",
+    "branding": false,
+    "appId": "5ecd978ef0aa2b501cf8cf25",
+    "controlsPosition": "top"
+  },
+});
+
+var headers = new Headers();
```

Comment:
essentially this is the exact same code that is in the index script, I followed how they had set it up in brownfield and it solved our issues